doc/submitter_guide.md

```diff
@@ -312,6 +312,94 @@ Date/Time Format][datetime_format].
 
 Example: `"2020-08-14T23:41:54+00:00"`
 
+#### Issues
+The following properties are used to describe issues found in reports
+
+##### `build_valid`
+The status to assign to incident builds
+
+Example: `"build_valid": false`
```

Comment:
It's enough to just show the value in the example, like the existing examples for all other properties. No need to also show the property name.

I.e. this:
```markdown
Example: `false`
```

And not this:
```markdown
Example: `"build_valid": false`
```

Here and everywhere else.